source/index.html.md

```diff
@@ -516,6 +516,14 @@ To simulate [transaction failures](#failure-reasons) create a Payment or Payment
     * Initiate a Payment Request for <code>$0.02</code>.
     * Split will mimic a failure to debit the contact's bank account.
 
+## NPP payment failures
+If you are utilising an [Account Float](https://help.split.cash/en/articles/4275280-utilising-an-account-float)  to create NPP payments, you can simulate a transaction that fails to process through the NPP channel by [creating a Payment from your account float](https://help.split.cash/en/articles/4275293-transacting-from-your-account-float) for one of the following amounts.
+
+| Transaction failure reason | Failure details | Amount |
+|----------------------------|-----------------|---------|
+| `refer_to_split`           | Real-time payment service currently unavailable |  $1.55  |
+| `refer_to_customer`        | Real-time payment rejected by recipient |  $1.60  |
+You will receive all the same notifications as if this happened in our live environment. We recommend you check out our article on [what happens when an NPP Payment fails](https://help.split.cash/en/articles/4405560-what-happens-when-an-npp-payment-fails)  to learn more about our what happens when an NPP Payment is unable to process.
```

Comment:
```suggestion
You will receive all the same notifications as if this happened in our live environment. We recommend you check out our article on [what happens when an NPP Payment fails](https://help.split.cash/en/articles/4405560-what-happens-when-an-npp-payment-fails)  to learn more about **our** what happens when an NPP Payment is unable to process.
```

tiny one > remove 'our'

Reply:
Cheers @sharmanat0r missed that typo haha. All fixed and I'll deploy it now